geoportal/geoportailv3_geoportal/scripts/finalize_c2c_23_data_adaptations.py

```diff
@@ -69,16 +69,16 @@ def main():
         r_area.name = 'restrictionarea-{}'.format(r_area.id)
         session.add(r_area)
 
-    # Insert group background (in GMF) if it not already exists.
-    treeitem = session.query(TreeItem).filter(TreeItem.name == 'background').one_or_none()
-    if treeitem is None:
-      layergroup = LayerGroup(
-        name='background',
-        is_expanded=False,
-        is_internal_wms=True,
-        is_base_layer=False
-      )
-      session.add(layergroup)
+    # Rename group bglayers to background (in GMF).
+    treeitem = session.query(TreeItem).filter(TreeItem.name == 'bglayers').one()
```

Comment:
Should we be able to run the script several times?
If yes it would be better to continue using `one_or_none`.

Reply:
Done. It's updated with one_or_none